src/OpenTelemetry/Internal/ConfigurationExtensions.cs

```diff
@@ -66,7 +66,9 @@ internal static class ConfigurationExtensions
 
         if (!Uri.TryCreate(stringValue, UriKind.Absolute, out value))
         {
-            throw new FormatException($"{key} environment variable has an invalid value: '{stringValue}'");
+            OpenTelemetrySdkEventSource.Log.InvalidEnvironmentVariable(key, stringValue);
+            value = default;
```

Comment:
Should `value` be touched here? (I guess no)

Reply:
good catch! it's an out param so it needs to be set, but it's already been set by the Try method in the if statement.